app/scripts/controllers/transactions/index.test.js

```diff
@@ -1740,6 +1740,9 @@ describe('Transaction Controller', function () {
             gas: '0x7b0d',
             gasPrice: '0x77359400',
           },
+          securityProviderResponse: {
+            flagAsDangerous: '0',
```

Comment:
should we update this to a number type, rather than a string type, to match with `app/scripts/controllers/transactions/index.js`?

same comment for below instances

Reply:
Good catch! 
I fixed it.